recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        return self.soup.find("ul", {"class": "recipe-specs"}).find_all("li")
+
+    def total_time(self):
+        time = self._specs()[0].find_next("span").get_text()
+        return get_minutes(re.sub(r" \S$| ", "", time))
```

Comment:
If I understand it correctly, this regex removes either the first space character found, or a single non-whitespace character at the end from the `time` text.  Are those required to handle some particular inputs?  (it isn't clear to me from the test case)

Reply:
`20 - 30 minutes` in different languages is the string this is attacking, and it removes anything after the last whitespace character - and it was also meant to remove any whitespace characters, but you're right, the latter part probably only removes the first whitespace so it's probably not needed. I'll do some testing.